scripts/jenkins/cloud/ansible/roles/list_packages/tasks/diff-packages.yml

```diff
@@ -0,0 +1,55 @@
+#
+# (c) Copyright 2019 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+---
+
+- name: Diff list of installed packages in plaintext
```

Comment:
The task didn't require the diff step, but I appreciate that you went the extra mile with it.

Reply:
This is what Victor started and got +- working. I just took over it. I know that it is not mentioned in the card. My bad I didn't read the card carefully when I took over it. :(